src/core/ReactCompositeComponent.js

```diff
@@ -123,7 +109,7 @@ var ReactCompositeComponentMixin = assign({},
     this._rootNodeID = null;
 
     this._instance.props = element.props;
-    this._instance.state = null;
+    // state gets set up during mount
```

Comment:
Er… never mind; the constructor calls getInitialState. This comment seems wrong then.